### DISCUSSION TITLE
How we run Open Community Working Meetings, a retro

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#279] [Governance and Policy]
I recently asked for some feedback on our OCWMs from people outside of our org/community and I have a number of suggestions to make based on those discussions.

While I want to present some suggestions, I also want to allow those who attend or would attend the meeting to provide feedback.
I feel like it would be most constructive if we were to follow a simple retrospective process.
I invite you to share your feedback in line with our Code of Conduct with reference to avoiding impersonal discussions via [BCP54](https://www.rfc-editor.org/rfc/rfc7154.html)(section 2.2). I hope we can focus on improvement rather than placing any blame.

I would like to run this retrospective by first allowing people to collect their thoughts and sharing them in public, and later during a meeting, blind vote on the top comments and suggestions.

The proposed template for collection thoughts is under two groups of headings...
Mad, Sad, Glad. And. Start, Stop, Continue.

Write 1-3 items for each heading, avoiding to much detail here. You will be invited to explain or expand on each item during the call, but not start a discussion so as to allow us to get through everyones items. (There will be time at the end of the call for further discussion, or discussion can be taken offline later.)

I'll let everyone know here ahead of time which meeting we will be discussing these comments.

We will look for common themes and the top voted themes or items and seek approaches to improve or remedy the concerns presented.

**If you'd rather provide this feedback in private, that's fine, and I'll field the feedback on our calls anonymously.**

**Comment by jdesrosiers** [comment]
I'm just going to throw out some observations. They can be categorized into the template later.

Generally, I think we do a good job of making sure everyone gets a chance to express their opinion.

We often see one person talking for long periods. This can make discussion more difficult and can make it harder to stay engaged and focused on topic.

We sometimes see people's comments going off on tangents. We need to do a better job at recognizing when that is happening and refocusing the conversation sooner. It might also be helpful to have a conversation about what tangents we've been hearing frequently and agree to avoid those topics unless they are on the agenda.

We often leave a discussion without coming to a resolution even if that resolution is that there is no resolution. We tend to just drift to the next topic.

**Comment by Relequestual** [comment]
After some discussions with some outside individuals, it has been suggested we should be taking actions during the call as opposed to relying on someone else to write them up after the call. This way we can make sure we are capturing actions with more accuracy and with agreeement of those on the call.
